README.md

```diff
@@ -1,68 +1,34 @@
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+# Welcome to my Portfolio
 
-## Available Scripts
+This project was made using React and Elastic UI as a learning practice.
 
-In the project directory, you can run:
+This is the latest version is deployed here: www.beatrizsanchez.dev
```

Comment:
`The latest version...`

Reply:
Change the address to `http://beatrizsanchez.dev` and make it a link

Reply:
Updated